src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -320,7 +330,8 @@ public static bool IsAppService(this IEnvironment environment)
         public static bool IsKubernetesManagedHosting(this IEnvironment environment)
         {
             return !string.IsNullOrEmpty(environment.GetEnvironmentVariable(KubernetesServiceHost))
-            && !string.IsNullOrEmpty(environment.GetEnvironmentVariable(PodNamespace));
+            && !string.IsNullOrEmpty(environment.GetEnvironmentVariable(PodNamespace))
```

Comment:
Update the existing IsKubernetesManagedHosting_ReturnsExpectedResult to test your new logic as well